go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/hashgraph/hedera-sdk-go/v2
 
-go 1.20
+go 1.21
+
+toolchain go1.23.2
```

Comment:
why change the go version?

Reply:
The version change was made during testing, but I can revert to the previous version if needed.

Reply:
Yes, please revert it.